projects/plugins/jetpack/modules/contact-form/grunion-contact-form.php

```diff
@@ -3955,11 +4002,12 @@ public function process_submission() {
 		 * @since 1.3.1
 		 *
 		 * @param string $message Feedback email message.
+		 * @param string $message Feedback email message as an array
```

Comment:
Should this be `array` then?

```suggestion
		 * @param array $message Feedback email message as an array
```